src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -139,7 +151,8 @@ private AssertionMessages() {
   protected String modalWrapper = "#WikiaConfirm";
   @FindBys(@FindBy(css = ".wds-banner-notification"))
   private List<WebElement> notificationElements;
-  @FindBy(css = "#WikiaArticle a[href*='Special:UserLogin']")
+  //TODO: Remove szl part of the selector when szl tests are not needed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)